pip/flatpak-pip-generator

```diff
@@ -38,6 +38,13 @@ parser.add_argument('--build-isolation', action='store_true',
                         'Mostly useful on pip that does\'t '
                         'support the feature.'
                     ))
+parser.add_argument('--ignore-installed',
+                    type=lambda s: s.split(","),
+                    default="",
+                    help="Comma-separated list of package names for which pip "
+                    "should ignore already installed packages. Useful when "
+                    "the package is installed in the SDK but not in the "
+                    "runtime.")
```

Comment:
Minor nit that this file mostly uses single quotes `'`.

Otherwise looks good to me.

Reply:
Good catch! I've changed it to single-quotes now.